show_api.md

```diff
@@ -0,0 +1,92 @@
+# Sparkle Motion Show API Proposal
+
+## Color
+
+We should probably define what a color is. Are we working in RGB space? ARGB?
+CMYK, hex, etc.?
```

Comment:
32 bits arranged ARGB feels pretty solid as the canonical representation (see `baaahs.Color`). We could have other classes for transient storage and calculations (e.g. `ColorHSV`).